WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/RemoteOrderSynchronizer.swift

```diff
@@ -203,8 +212,10 @@ private extension RemoteOrderSynchronizer {
             .share()
             .eraseToAnyPublisher()
 
-        bindOrderCreation(trigger: syncTrigger)
-        bindOrderUpdate(trigger: syncTrigger)
+        if case .creation = flow {
+            bindOrderCreation(trigger: syncTrigger)
+        }
```

Comment:
Can this be `if flow == .creation`?

Reply:
Agree, simpler comparison is easier to read.
Updated in 6b0cb6d.